docs/products/postgresql/howto/pgaudit-logging.rst

```diff
@@ -0,0 +1,110 @@
+Use PGAudit for logging on Aiven for PostgreSQL®
+================================================
+
+Discover the PostgreSQL® audit logging extension and its capabilities. Learn how to enable PGAudit on your Aiven for PostgreSQL service and how to access and visualize your logs.
+
+About PGAudit
+-------------
+
+`PGAudit <https://www.pgaudit.org/#>`_ is a PostgreSQL extension that allows comprehensive session and object audit logging. PGAudit is particularly useful when you need detailed logs for state, financial, or ISO certification audits, where *audits* are official inspections conducted by independent institutions authorized to examine specific subjects.
+
+What distinguishes PGAudit from other logging facilities is the level of detail it supports for logs. With a regular logging tool, you can get a list of all the operations performed against the database, which is good enough for monitoring purposes. However, audits often require getting to specific statements as requested by auditors, which is what PGAudit helps you achieve.
+
+Set up PGAudit
+--------------
+
+Prerequisites
+'''''''''''''
+
+* Aiven for PostgreSQL version 11 or higher
+* ``avnadmin`` role for enabling and configuring PGAudit
+* :doc:`Aiven CLI </docs/tools/cli>` / ``psql``
+
+Enable audit logs
+'''''''''''''''''
+
+To configure PGAudit, use the ``aiven-extras`` extension and its ``set_pgaudit_parameter()`` function on a per-database level.
```

Comment:
**suggestion**: I think its important to mention that this configurations will take effect only on new connections. We already had an [issue](https://aiven.atlassian.net/browse/BF-1893?atlOrigin=eyJpIjoiN2I5MmI0MjZmNGIxNDQ4YWI5NWI3MWFiNmEzZjExNGIiLCJwIjoiaiJ9) where a client reported they couldn't see any changes.

Reply:
@kathia-barahona , this is a valid point, thank you!

Reply:
@kathia-barahona I am wondering do we have an issue with pgbouncer then in cases where it's used.